test/DynamoCoreUITests/RecordedTests.cs

```diff
@@ -684,10 +689,22 @@ protected void LoadCustomNodes(string[] customNodeFilePaths)
                 throw new InvalidOperationException(message);
             }
 
+            var geometryFactoryPath = string.Empty;
+            if (preloadGeometry && (preloader == null))
```

Comment:
Just noticing the special care to check if preloader == null.  

Does `Preloader` prevent duplicate calls in a static way?  I would prefer to not have special code in here to prevent multiple invocations.  Instead, that should live in the original code and we should call Preload as many times as we please (assuming that internally it will prevent duplicate loads).